src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    divide(a, b) {
+      return a / b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
```

Comment:
pass just value from the operate arguments (num), no need to pass the result, you can get it inside the operation itself